src/_posts/2014-02-19-appveyor-20-dedicated-build-vms-parallel-testing-nuget-deployment.md

```diff
@@ -23,7 +23,7 @@ deployment.
 * [NuGet hosting](#nuget)
 * [Build matrix](#build-matrix)
 * [Parallel testing](#parallel-testing)
-* [Deployment to multiple environments](#deployment)
+* [Deployment to multiple environments](#deploying)
```

Comment:
I see now. It's a bug with the paginator plugin, I believe, since we have the same issue with /updates/.

There, I just ignore it from the html linter. Perhaps I should do the same here.

For reference https://github.com/appveyor/website/blob/master/Gruntfile.js#L270